src/main/javatests/com/google/summit/translation/ClassDeclarationTest.kt

```diff
@@ -100,32 +98,37 @@ class ClassDeclarationTest {
         }
         """
 
-    val fieldDecl = TranslateHelpers.parseAndFindFirstNodeOfType<FieldDeclaration>(input)
+    val fieldDeclGroup = TranslateHelpers.parseAndFindFirstNodeOfType<FieldDeclarationGroup>(input)
 
-    assertNotNull(fieldDecl)
+    assertNotNull(fieldDeclGroup)
+    assertThat(fieldDeclGroup.declarations).hasSize(1)
+    val fieldDecl = fieldDeclGroup.declarations.first()
     assertThat(fieldDecl.qualifiedName).isEqualTo("Test.field")
     assertThat(fieldDecl.modifiers).hasSize(1)
     assertThat(fieldDecl.hasKeyword(KeywordModifier.Keyword.PUBLIC)).isTrue()
     assertThat(fieldDecl.initializer).isNotNull()
   }
 
   @Test
-  fun multipleFieldDeclarators_translates_toMultipleFieldDeclarations() {
+  fun multipleFieldDeclarators_translate_toFieldDeclarationGroups() {
     val input =
       """
         class Test {
           public Int field1 = 1, field2 = 2;
+          public Int field3 = 3;
```

Comment:
Great test: to clarify whether single fields are still in a `FieldDeclarationGroup`.